classes/ai_access_handler.php

```diff
@@ -0,0 +1,102 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace quizaccess_ai;
+
+use local_ai_manager\ai_manager_utils;
+use stdClass;
+
+/**
+ * Helper class that encapsulates the AI availability check for quizaccess_ai.
+ *
+ * @package   quizaccess_ai
+ * @copyright 2025, ISB Bayern
```

Comment:
```suggestion
 * @copyright 2025 ISB Bayern
```

pls also fix at all other places

Reply:
Fixed in all files